tidy3d/components/simulation.py

```diff
@@ -1592,14 +1569,9 @@ def get_eps(medium: Medium, freq: float):
         def make_eps_data(coords: Coords):
             """returns epsilon data on grid of points defined by coords"""
             xs, ys, zs = coords.x, coords.y, coords.z
-            rmin = tuple(coord[0] for coord in (xs, ys, zs))
-            rmax = tuple(coord[-1] for coord in (xs, ys, zs))
-            points_box = Box.from_bounds(rmin=rmin, rmax=rmax)
             x, y, z = np.meshgrid(xs, ys, zs, indexing="ij")
             eps_array = eps_background * np.ones(x.shape, dtype=complex)
             for structure in self.structures:
-                if not points_box.intersects(structure.geometry):
-                    continue
```

Comment:
Why was this removed? Is it some weird mix up in creating the geo_group PR before this was added? But I'm confused because even after rebasing against develop, it is still missing (so if we merge this PR it will remove it).

Reply:
I think this was a mistake, maybe it's because I made this branch before we started skipping non-intersecting boxes?

Reply:
we can apply this commit to the PR?
https://github.com/flexcompute/tidy3d/commit/9f7cf740119720a82fee1393e90b4611b3e71d23

Reply:
Yeah ok I was just checking that there wasn't anything intentional here. I'll fix it.